lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -954,7 +954,12 @@ def headers(options)
         cookie = options[:cookie] || cookie
         headers['Cookie'] = cookie if cookie
 
-        headers['Idempotency-Key'] = idempotency_key if idempotency_key
+        # Required due Worldpay does not accept duplicated idempotency keys in different transactions
+        # which would be the case for authorize => capture
```

Comment:
I would rewrite this so it's more clear
```suggestion
        # Required because Worldpay does not accept duplicate idempotency keys 
        # for different transactions, such as in the case of an authorize => capture flow.
```

Reply:
Thanks for the suggestion! Change added